tripy/tests/spec_verification/test_dtype_constraints.py

```diff
@@ -17,118 +17,188 @@
+                           "permute": {"perm": {"init": (1, 0)}},
+                           "quantize": {"scale": {"init": [1, 1, 1]}, "dim": {"init": 0}},
+                           "sum": {"dim": {"init": 0}},
+                           "all":{"dim": {"init": 0}},
+                           "any": {"dim": {"init": 0}},
+                           "max": {"dim": {"init": 0}},
+                           "prod": {"dim": {"init": 0}},
+                           "mean": {"dim": {"init": 0}},
+                           "var": {"dim": {"init": 0}},
+                           "argmax": {"dim": {"init": 0}},
+                           "argmin": {"dim": {"init": 0}},
+                           "reshape": {"shape": {"init": (6,)}},
+                           "squeeze": {"input": {"shape": (3, 1)}, "dims": {"init": (1)}},
+                           "__getitem__": {"index": {"init": 2}},
+                           "split": {"indices_or_sections": {"init": 2}},
+                           "unsqueeze": {"dim": {"init": 1}},
+                           "masked_fill": {"value": {"init": 1}},
+                           "ones": {"shape": {"init": (3,2)}},
+                           "zeros": {"shape": {"init": (3,2)}},
+                           "arange": {"start": {"init": 0}, "stop": {"init": 5}},
+                          }
+
+# Add default_constraints to input_values within TYPE_VERIFICATION
+for func_name, (func_obj, input_dict, _, _, types_assignments) in TYPE_VERIFICATION.items():
+    default_constraints = default_constraints_all.get(func_name, None)
+    if default_constraints != None:
+        for param_name, input_info in input_dict.items():
+            input_values = list(input_dict[param_name].values())[0]
+            other_constraint = default_constraints.get(param_name, None)
+            if other_constraint:
+                for key, val in other_constraint.items():
+                    if key == "init":
+                        input_values.init = val
+                    elif key == "shape":
+                        input_values.shape = val
+                    elif key == "target":
+                        input_values.target = val
+                    elif key == "count":
+                        input_values.count = val
+                    else:
+                        raise RuntimeError(f"Could not match key for default_constraints. Key was {key}, value was {val}")
 
 func_list = []
-for func_obj, parsed_dict, types_assignments in TYPE_VERIFICATION.values():
-    inputs = parsed_dict["inputs"]
-    returns = parsed_dict["returns"]
-    types = parsed_dict["types"]
-    # exclude quantization types until q/dq MR is merged
+for func_name, (func_obj, inputs, return_dtype, types, types_assignments) in TYPE_VERIFICATION.items():
+    # Issue #268 exclude float8 until casting to float8 gets fixed.
+    # Issue #268 exclude int4 until int4 is representable.
```

Comment:
So that we don't miss this:
```suggestion
    # TODO (#268) exclude float8 until casting to float8 gets fixed.
    # TODO (#268) exclude int4 until int4 is representable.
```